### PR TITLE
Apply formatting

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
This PR apply prettier formatting to all files, according to TS Coding Conventions. Now that we have auto format on commit, this is one time format to reduce unnecessary changes in new PRs.
Also adds 10 more seconds after graph node start, to allow enough time before creating subgraph.
Also attempt to fix chart install workflow and dapp tests.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by AlfredoG87 — DISMISSED** [review]
I am afraid this will break and ask for rebase of every single outstanding PRs but nevertheless worth to have conventions implemented on the project 👍

**Comment by georgi-l95** [comment]
@AlfredoG87 i've rebased and lost approval

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "singleQuote": true,
+    "singleQuote": false,
```
Comment (Nana-EC) [inline .prettierrc:2]: I prefer single actually 😆

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Thanks.
Blocking whiles I confirm some items

**Review by ebadiere — APPROVED** [review]
Mainly single quotes to double quotes, added semi-colons and some method call formatting.  LGTM.

File: .prettierrc (hunk 1 above)
Comment (georgi-l95) [reply .prettierrc]: Okay, done, switched to single :D

**Pull request merged**